attributes/monitors.rb

```diff
@@ -5,39 +5,39 @@
 # port monitor for eleastic search http
 # this port is not usually publicly accessible, disable by default
 cmd['port_9200']['disabled'] = true
-cmd['port_9200']['period'] = '60'
-cmd['port_9200']['timeout'] = '30'
```

Comment:
It feels like these should be strings, unless we plan to do arithmetic on them. Maybe we can fix that in rackspace_monitoring.